packages/flutter/lib/src/widgets/semantics_debugger.dart

```diff
@@ -76,6 +76,8 @@ class _SemanticsDebuggerState extends State<SemanticsDebugger> with WidgetsBindi
         });
       }
     });
+    // addPostFrameCallback does *not* schedule a frame.
+    SchedulerBinding.instance.scheduleFrame();
```

Comment:
addPostFrameCallback doesn't schedule a frame, but we're calling it from within a frame, and in the callback we're calling setState, and setState _does_ schedule a frame, so I don't understand why this makes a difference?